autoload/neomake/makers/ft/vhdl.vim

```diff
@@ -0,0 +1,16 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#vhdl#EnabledMakers()
+    if executable('ghdl')
+        return ['ghdl']
+    else
+        return []
+    endif
+endfunction
+
+function! neomake#makers#ft#vhdl#ghdl()
+    return {
+                \ 'args' : ['-s'],
+                \ 'errorformat' : '%f:%l:%c: %m',
```

Comment:
Please include the error type here.
Use `%E` in the beginning if all entries are errors.